openslides/assignments/models.py

```diff
@@ -366,6 +366,7 @@ class AssignmentPoll(RESTModelMixin, CollectDefaultVotesMixin,
         related_name='polls')
     yesnoabstain = models.BooleanField(default=False)
     yesno = models.BooleanField(default=False)
+    without_abstains = models.BooleanField(default=False)
```

Comment:
Sorry. I though we take an aproach without new database values. The percent base is just calculated on the fly if config and situation matches.